tests/unit/core/test_views.py

```diff
@@ -11,17 +11,6 @@ def setUp(self):
         self.viewset = views.ProjectorViewSet()
         self.viewset.format_kwarg = None
 
-    def test_activate_elements(self, mock_object):
```

Comment:
Please (if there is no time ti fix the tests) only disable them and add an issue that we can fix them later.

Reply:
The problems is the mock object. I would suggest, that we no not use unit tests for views anymore.